src/Symfony/Component/Stopwatch/Tests/StopwatchTest.php

```diff
@@ -71,11 +69,12 @@ public function testStop()
     {
         $stopwatch = new Stopwatch();
         $stopwatch->start('foo', 'cat');
-        usleep(200000);
+        usleep(100000);
         $event = $stopwatch->stop('foo');
-
+        $duration = $event->getDuration();
+        usleep(100000);
         $this->assertInstanceof('Symfony\Component\Stopwatch\StopwatchEvent', $event);
-        $this->assertEquals(200, $event->getDuration(), null, self::DELTA);
+        $this->assertEquals($duration, $event->getDuration());
```

Comment:
If I always return ``0`` or ``null`` in ``getDuration`` (removing the return statement for instance), this test will pass. This means that it is not testing the method anymore